docs/source/guides/understanding_types_and_tags.ipynb

```diff
@@ -46,7 +46,9 @@
     "* `string`\n",
     "* `timedelta64[ns]`\n",
     "\n",
-    "The physical type conversion is done based on the `LogicalType` that has been specified or inferred for a given column."
+    "The physical type conversion is done based on the `LogicalType` that has been specified or inferred for a given column.\n",
+    "\n",
+    "When using Woodwork with a Koalas DataFrame, physical types different than those listed above may be used. For more information, refer to the guide [Using Woodwork with Dask and Koalas DataFrames](https://woodwork.alteryx.com/en/stable/guides/using_woodwork_with_dask_and_koalas.html#Notes-on-Koalas-Dtype-Conversions)."
```

Comment:
missing a word - "...Koalas DataFrame, physical types **are/may be** different than those..."

Reply:
Updated wording of this sentence.